### DISCUSSION TITLE
Env format support and the default format of code blocks

### DISCUSSION THREAD

**Discussion by 1itt1eB0y** [outline/outline#3789] [Ideas]
the code blocks lacks support of the ENV file. I'm really expect it.
And the default format always JS, why not make it to remember the format you last chose and still use that one, or just like "```json", "```yaml"

**Comment by tommoor** [comment]
> why not make it to remember the format you last chose

It's supposed to do this, so it sounds like could file a bug.

**Reply by 1itt1eB0y** [reply]
I will try to check it later
